setup.py

```diff
@@ -16,7 +16,7 @@
     url='https://github.com/openfisca/openfisca-country-template',
     include_package_data = True,  # Will read MANIFEST.in
     install_requires=[
-        'OpenFisca-Core >= 23, < 24.0',
+        'OpenFisca-Core >= 23.1, < 24.0',
```

Comment:
> Any reason for that change here or is it just a knight? 😛

Reply:
The changelog claims that we are making this package compatible with python 3.
We thus need to make sure we request a core version that is compatible with python 3. Actually, 23.1.1 may be even better.